cves/CVE-2017-16840.yml

```diff
@@ -344,7 +370,14 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    In my opinion the vulnerablity arose because of a lack of testing and that 
+    one developer was responsible for those files. According to the commits, one developer 
+    wrote the classes where the vulnerability existed and it had been edited by a few 
+    other people (including the developer). As for the lack of testing, it was identified 
+    that the vulernability was because the was not proper padding on all sides of the frame 
+    but rather just the bottom and left. These claims are supported by the fact that the 
+    time between the vulnerability and fixed was less than a month.
```

Comment:
Could it have been due to a lack of understanding of specification requirements? See the fix git log.